crates/oxc_linter/src/snapshots/eq_eq_eq.snap

```diff
@@ -0,0 +1,33 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: eq_eq_eq
+---
+
+  ⚠ eslint(eqeqeq): Require the use of === and !==
+   ╭─[eq_eq_eq.tsx:1:1]
+ 1 │ a == b
+   · ──────
+   ╰────
+  help: Prefer strict (in)equality operator
```

Comment:
It would be nice to suggest to use `===` and `!==`.

"Prefer the strict `===` operator".
"Prefer the strict `!==` operator".

Reply:
I wonder if we can hack it and find the span on the operator ... 

we can get the source text, get the string slice between the two expressions and then find the span for the operator.

Reply:
Yep I think we can improve the messaging here. Will fix it up

Reply:
Added better errors 👍🏼